frontend/src/i18n/locales/translation.interface.ts

```diff
@@ -0,0 +1,9 @@
+interface Translation {
```

Comment:
supernit: prefer `types.ts` over `translation.interface.ts` for consistency with rest of frontend. If we really want to use dot notation, maybe `translation.types.ts`

Reply:
done